database/seeds.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO department (name)
```

Comment:
Nice job including a seed file! This makes it super easy for other users to get started using your application right away :)